src/controllers/dlgprefcontroller.cpp

```diff
@@ -311,15 +312,21 @@ void DlgPrefController::enumeratePresets(const QString& selectedPresetPath) {
 
     // qDebug() << "Enumerating presets for controller" << m_pController->getName();
 
+    // Check the text color of the palette for whether to use dark or light icons
+    QDir iconsPath = QString(":/images/preferences/dark/");
+    if (!Color::isDimColor(palette().text().color())) {
+        iconsPath = QString(":/images/preferences/light/");
```

Comment:
```
home/jan/Projects/mixxx/src/controllers/dlgprefcontroller.cpp: In member function ‘void DlgPrefController::enumeratePresets(const QString&)’:
/home/jan/Projects/mixxx/src/controllers/dlgprefcontroller.cpp:318:58: warning: ‘QDir& QDir::operator=(const QString&)’ is deprecated: Use QDir::setPath() instead [-Wdeprecated-declarations]
  318 |         iconsPath = QString(":/images/preferences/light/");
      |                                                          ^
In file included from /usr/include/qt/QtCore/QDir:1,
                 from /home/jan/Projects/mixxx/src/controllers/controllerpreset.h:4,
                 from /home/jan/Projects/mixxx/src/controllers/controllermappingtablemodel.h:10,
                 from /home/jan/Projects/mixxx/src/controllers/controllerinputmappingtablemodel.h:8,
                 from /home/jan/Projects/mixxx/src/controllers/dlgprefcontroller.h:6,
                 from /home/jan/Projects/mixxx/src/controllers/dlgprefcontroller.cpp:1:
/usr/include/qt/QtCore/qdir.h:110:11: note: declared here
  110 |     QDir &operator=(const QString &path);
      |           ^~~~~~~~
```

Reply:
Thank you. Locally I still build with Qt5.12.8 (version in Ubuntu 20.04) and with that I don't see deprecation warnings.
Changed to what you literally suggested above.